pkg/services/ngalert/store/alertmanager.go

```diff
@@ -76,24 +76,36 @@ func (st DBstore) SaveAlertmanagerConfigurationWithCallback(ctx context.Context,
 			OrgID:                     cmd.OrgID,
 			CreatedAt:                 time.Now().Unix(),
 		}
+
 		// TODO: If we are more structured around how we seed configurations in the future, this can be a pure update instead of upsert. This should improve perf and code clarity.
 		upsertSQL := st.SQLStore.GetDialect().UpsertSQL(
 			"alert_configuration",
 			[]string{"org_id"},
 			[]string{"alertmanager_configuration", "configuration_version", "created_at", "default", "org_id", "configuration_hash"},
 		)
-		params := append(make([]interface{}, 0), cmd.AlertmanagerConfiguration, cmd.ConfigurationVersion, config.CreatedAt, config.Default, config.OrgID, config.ConfigurationHash)
-		if _, err := sess.SQL(upsertSQL, params...).Query(); err != nil {
-			return err
-		}
+		params := append(
+			make([]interface{}, 0),
+			cmd.AlertmanagerConfiguration,
+			cmd.ConfigurationVersion, config.CreatedAt,
```

Comment:
We should do each parameter on a new line, or all on one line - not a mixture of both